inc/cropper/namespace.php

```diff
@@ -488,3 +507,370 @@ function prevent_thumbnail_generation( $sizes ) {
 
 	return [];
 }
+
+/**
+ * Fake attachment meta data to include all image sizes.
+ *
+ * This attempts to fix two issues:
+ *  - "new" image sizes are not included in meta data.
+ *  - when using Tachyon in admin and disabling resizing,
+ *    NO image sizes are included in the meta data.
+ *
+ * @TODO Work out how to name files if crop on upload is reintroduced.
+ *
+ * @param $data          array The original attachment meta data.
+ * @param $attachment_id int   The attachment ID.
+ *
+ * @return array The modified attachment data including "new" image sizes.
+ */
+function filter_attachment_meta_data( $data, $attachment_id ) {
+	// Save time, only calculate once.
+	static $cache = [];
+
+	if ( isset( $cache[ $attachment_id ] ) ) {
```

Comment:
```suggestion
	if ( empty( $cache[ $attachment_id ] ) ) {
```

Nit: unlikely to make a difference